docs/jade-getting-started.md

```diff
@@ -262,6 +265,21 @@ create extension pgcrypto;
   * Google Project Name: `broad-jade-ZZ` (replacing `ZZ` with your initials)
   * Google Organization: broadinstitute.org
 
+6. `ctrl-d` out of the psql shell and run the following to connect to your personal instance database:
+
+```
+cd jade-data-repo/ops
+DB=datarepo-ZZ SUFFIX=ZZ ENVIRONMENT=dev ./db-connect.sh
+```
+
+When in the psql shell again, run:
+
+```
+insert into billing_profile
+values ('390e7a85-d47f-4531-b612-165fc977d3bd', 'default', 'direct', '<BILLING_PROFILE>', null, '11/6/2020 8:29:59 AM', null);
+```
+replacing `<BILLING_PROFILE>` with the value at `vault read -field=billing_account secret/dsde/datarepo/dev/billing_account`
```

Comment:
You could make this even simpler to follow by making a step before this one to run this command before running db-connect.